.gitignore

```diff
@@ -38,6 +38,8 @@ ironic-deployment/overlays/temp
 
 # Development containers (https://containers.dev/)
 .devcontainer
+# Python venv
+.venv
```

Comment:
q: There isn't any python files in the repo, why do we need to ignore `.venv` ?

Reply:
Ironic and related things like the `baremetal` CLI is all Python. Sometimes I find it useful to set up a venv for installing these. The CLI is especially useful to check what state Ironic has.

Reply:
This is fine as well, but in general I'd advise using `virtualenvwrapper` and set its root to `~/.config/venvs` or similar instead of "polluting" repos. `workon` is super handy for switching between venvs.

Reply:
Thanks for the advice! Maybe I need to adopt this. I have been going back and forth, also using `pipenv`, but then revert to `venv` since that is "always" there.